### PR TITLE
add consensus banner

### PR DESCRIPTION
**Pull request by carinaluise**
task: https://app.asana.com/0/1204010243239588/1204310044597580/f

### CODE REVIEW

File: polkadot-wiki/docusaurus.config.js [hunk 1]
```diff
@@ -248,14 +248,14 @@ module.exports = {
...
     /* Banner / Announcement bar */
-    // announcementBar: {
-    //   id: "banner",
-    //   content:
-    //   'Join Polkadot at <strong>SXSW</strong> to discover the true potential of Web3. <a href="https://polkadot.network/ecosystem/events/sxsw-2023/?utm_source=wiki.polkadot.network&utm_medium=referral&utm_campaign=sxsw%2023&utm_content=notification" target="_blank" rel="noopener nofollow noreferrer">Learn More &rarr;</a>',
-    //   backgroundColor: '#e6007a',
-    //   textColor: 'white',
-    //   isCloseable: true,
-    // },
-    image: 'img/og-polkadot.png'
```
Comment (profradha) [inline polkadot-wiki/docusaurus.config.js:259]: Why is this removed? This was added earlier today https://github.com/mmurakaru/polkadot-wiki/pull/1

**Review by DrW3RK — APPROVED** [review]
